### ISSUE TITLE
TypeError: __init__() missing 1 required positional argument: 'on_delete'

### ISSUE THREAD

**Issue by xhriscortz** [furious-luke/django-address#80]
from django.db import models
from django.contrib.auth.models import User

# Create your models here.
class Game(models.Model):
    first_player = models.ForeignKey(User,
                                     related_name="games_first_player")
    second_player = models.ForeignKey(User,
                                      related_name="games_second_player")

    start_time = models.DateTimeField(auto_now=True)
    last_active = models.DateTimeField(auto_now=True)

class Move(models.Model):
    x = models.IntegerField()
    y = models.IntegerField()
    comment = models.CharField(max_length=300, blank=True)
    by_first_player = models.BooleanField()
    game = models.ForeignKey('Game',
                             on_delete=models.CASCADE)

All the tutorials I've seen have directed me to add on_delete for the ForeignKey, but that's what I had at first when I initially got the problem. Can someone show me another way.

**Comment by n6151h** [comment]
If you're using ``AddressField`` in your model, that's where your problem's coming from.  This is easily fixed by adding
```
kwargs['on_delete'] = kwargs.get('on_delete', models.CASCADE)
```
after line 309 in address/models.py

**Comment by thiras** [comment]
#81 already waiting. Please merge @furious-luke

**Comment by banagale** [comment]
I looked at this. I do not believe that it is normal for a modelfield's to subclass model.foreignkey. Here's [docs on custom modelfields](https://docs.djangoproject.com/en/3.0/howto/custom-model-fields/).

That said, if an error is still popping out and this resolves it, it should be fixed in 0.2.2 or 0.2.3.  Assigning this to 0.2.2 for now.

**Comment by banagale** [comment]
This was required to pass tests. Merged in #28 
Thank you to @thiras, @n6151h  and @xhriscortz for contributing to this issue.
